src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
```

Comment:
I would rename this to `DeleteAvailabilityServlet.java`

Reply:
That sounds good thanks!